integration/cmd_app_test.go

```diff
@@ -23,7 +24,40 @@ func TestGenerateAnAppAndVerify(t *testing.T) {
 	env.EnsureAppIsSteady(path)
 }
 
-func TestGenerateAnAppWithWasmAndVerify(t *testing.T) {
+func TestGenerateAnAppWithNoDefaultModule(t *testing.T) {
```

Comment:
I think we should check presence of the custom module dir at $path/x/blog, what do you think?